packages/browser/src/sdk.ts

```diff
@@ -166,3 +174,54 @@ export function close(timeout?: number): PromiseLike<boolean> {
 export function wrap(fn: (...args: any) => any): any {
   return internalWrap(fn)();
 }
+
+/**
+ * Enable automatic Session Tracking for the initial page load.
+ */
+function startSessionTracking(): void {
+  const window = getGlobalObject<Window>();
+  const hub = getCurrentHub();
+  let fcpResolved = false;
+  let loadResolved = false;
+  const possiblyEndSession = (): void => {
+    if (fcpResolved && loadResolved) {
+      hub.endSession();
+    }
+  };
+
+  hub.startSession();
+
+  window.addEventListener('load', () => {
+    loadResolved = true;
+    possiblyEndSession();
+  });
```

Comment:
I think we need to account for the page being already loaded when the `startSessionTracking` function runs? Otherwise the event may never fire and we'd miss a session.

Like in https://github.com/GoogleChrome/web-vitals/blob/949744395cd281483bdfa9f3a663138fb55d4a5e/test/utils/imagesPainted.js#L25-L31.

Reply:
Good catch, updated.